### ISSUE TITLE
never hit 100%

### ISSUE THREAD

**Issue by jimblue** [clessg/progress-bar-webpack-plugin#19]
Just to know if it's normal that the progress bar never hit 100%?
It's kind of disturbing, because build is completed at 100% in reality.

```bash
> babel-node webpack/start-app.js

  BUILD ███████████████████░░█ 91% (6.4 seconds)

 DONE  Compiled successfully in 14379ms                                                                                             5:23:12 PM

Build completed in 13.059s
```

**Comment by adamzasah** [comment]
It bugs me too.

**Comment by huruji** [comment]
I fixed, see [https://github.com/clessg/progress-bar-webpack-plugin/pull/29](https://github.com/clessg/progress-bar-webpack-plugin/pull/29) @jimblue @adamzasah @clessg

**Comment by kamilbrk** [comment]
This broke the `:msg` variable

**Comment by clessg** [comment]
Thanks @kamilbrk, I've fixed that in 1.12.1.
